### DISCUSSION TITLE
Where are the relevance scores of eval queries in the MSMARCO?

### DISCUSSION THREAD

**Discussion by miguelwon** [castorini/anserini#2466] [General]
Hi, looking for some expert help on the MSMARCO.

I'm getting familiar with the dataset, particular because I'm working with passage IR.
For that I'm reading your excellent  guide [start here](https://github.com/castorini/anserini/blob/master/docs/start-here.md), but I'm still a bit confused about the files `queries.eval.tsv` or `queries.eval.small.tsv`. Where are the relevance scores of these files?  

Thanks!

**Comment by lintool** [comment]
Hi @miguelwon not sure I understand your question. The qrels tell you if a docid is relevant (or not) wrt a query. With `trec_eval`, you compute metrics like MRR, nDCG, etc. Does this help?

**Reply by miguelwon** [reply]
Hi and thanks for your answer. Sorry, I was not clear with my question.
What I would like to know is if the file for the relevance scores for eval queries are public somewhere.

**Reply by lintool** [reply]
They are here: https://github.com/castorini/anserini-tools/tree/master/topics-and-qrels begins with `qrels.*`

**Reply by miguelwon** [reply]
Thanks!
